src/frontend/ExtensionManager/components/ExtensionContents/index.tsx

```diff
@@ -22,36 +24,47 @@ const animation = {
   transition: { duration: 0.2 }
 }
 
+export function ExtensionPopup() {
+  return (
+    <webview
+      nodeintegrationinsubframes="true"
+      webpreferences="contextIsolation=true, nodeIntegration=true"
+      src={`chrome-extension://${extensionState.extensionId}/popup.html`}
+      className={ExtensionContentsStyles.mmWindow}
+      allowpopups={trueAsStr}
+    ></webview>
+  )
+}
+
+export function ExtensionNotification() {
+  return (
+    <webview
+      nodeintegrationinsubframes="true"
+      webpreferences="contextIsolation=true, nodeIntegration=true"
+      src={`chrome-extension://${extensionState.extensionId}/notification.html`}
+      className={ExtensionContentsStyles.mmWindow}
+      allowpopups={trueAsStr}
+    ></webview>
+  )
+}
```

Comment:
could maybe combine these and pass in the `notification.html` or `popup.html` to keep it dry

Reply:
I think the code duplication is fine in this case